app/controllers/ctc/questions/claim_eitc_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Ctc
+  module Questions
+    class ClaimEitcController < QuestionsController
+      include Ctc::ResetToStartIfIntakeNotPersistedConcern
+
+      layout "yes_no_question"
+
+      def self.show?(intake)
+        Flipper.enabled?(:eitc) && !intake.home_location_puerto_rico?
```

Comment:
i think technically it might be better if we used intake.puerto_rico_filing? here, since that also includes the feature flag and could potentially include other things that make something a puerto rico filing in the future... but also maybe not. im not sure which is better. my preference would be to change it to intake.puerto_rico_filing? though.